spec/list-actions.lsl

```diff
@@ -10,6 +10,13 @@ define action LIST_COPY(
         to: int
     ): list<any>;
 
+define action LIST_TO_ARRAY(
```

Comment:
There should be a comment on why there are both *input* and *output* arrays.

Reply:
@dog-m In such way: 
`// We have input and output arrays, because we have method with such signature: public <T> T[] toArray(T[] a) `

?

Reply:
No, at least something like:
```java
// Copies contents of the list into the provided array. Allows auto-resizing.
// Example behavior from Java: String[] x = arrList.toArray(new String[0]);
```

Reply:
Fixed